cluster/images/etcd/Makefile

```diff
@@ -17,9 +17,10 @@
 # Usage:
 # 	[TAG=2.2.1] [REGISTRY=gcr.io/google_containers] [ARCH=amd64] [BASEIMAGE=busybox] make (build|push)
 
-TAG?=3.0.4
+ETCD_TAG?=3.0.4
+TAG?=3.0.4-migration.5
 ARCH?=amd64
-REGISTRY?=gcr.io/google_containers
+REGISTRY?=gcr.io/groovy-sentry-504
```

Comment:
suggest you set non-default values via `make blah REGISTRY=gcr.io/groovy-sentry-504 TAG=3.0.4-migration.5` instead of editing this file :)